roles/maintenance_32_apache/tasks/main.yml

```
@@ -0,0 +1,278 @@
+---
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignore-me
+    name: bar
+
+# Include OS specific vars to get binary name: apache_binary_name, httpd or apache2
+- <<: *task
+  vars:
+    taskid: 32-009
+    name: "Apache: Gather details, set vars: set apache binary based on OS"
```

Not an actual checklist task:

```suggestion
- name: "Apache: Gather details, set vars: set apache binary based on OS"
```